apps/frontend/src/pages/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,152 @@
+import { Button, Loader, ProgressBar } from "~/libs/components/components.js";
+import { PREVIOUS_INDEX_OFFSET } from "~/libs/constants/constants.js";
+import {
+	useAppDispatch,
+	useAppForm,
+	useAppSelector,
+	useCallback,
+	useEffect,
+	useState,
+} from "~/libs/hooks/hooks.js";
+import {
+	actions as onboardingActions,
+	oneAnswerSelectedValidationSchema,
+} from "~/modules/onboarding/onboarding.js";
+import { ONBOARDING_FORM_DEFAULT_VALUES } from "~/pages/onboarding/libs/constants/constants.js";
+
+import { OnboardingAnswer } from "./libs/components/onboarding-answer/onboarding-answer.js";
+import {
+	type OnboardingAnswerRequestBodyDto,
+	type OnboardingFormValues,
+} from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+const Onboarding: React.FC = () => {
+	const dispatch = useAppDispatch();
+
+	const [answerIds, setAnswerIds] = useState<OnboardingAnswerRequestBodyDto>({
+		answerIds: [],
+	});
+
+	const {
+		currentQuestionIndex,
+		isLastQuestion,
+		question,
+		totalQuestionsAmount,
+	} = useAppSelector(({ onboarding }) => ({
+		currentQuestionIndex: onboarding.currentQuestionIndex,
+		isLastQuestion:
+			onboarding.currentQuestionIndex ===
+			onboarding.questions.length - PREVIOUS_INDEX_OFFSET,
+		question: onboarding.currentQuestion,
+		totalQuestionsAmount: onboarding.questions.length,
+	}));
+
+	useEffect(() => {
+		void dispatch(onboardingActions.getAll());
+	}, [dispatch]);
+
+	const { control, handleSubmit, isValid, reset } =
+		useAppForm<OnboardingFormValues>({
+			defaultValues: ONBOARDING_FORM_DEFAULT_VALUES,
+			validationSchema: oneAnswerSelectedValidationSchema,
+		});
+
+	const handleSaveAnswers = useCallback(
+		(payload: OnboardingAnswerRequestBodyDto) => {
+			// void dispatch(onboardingActions.saveAnswers(payload));
+			return payload;
```

Comment:
```suggestion
```

Reply:
I need to use `return payload`; to avoid linter errors, or I need to remove `answerIds` using useState (like declared but never used)